### DISCUSSION TITLE
Stuck!!!

### DISCUSSION THREAD

**Discussion by EdmundDantes1** [community/community#148126] [New to GitHub]
is:pr is giving me red wavy lines on intro to GitHub & won't let me progress. 
![image](https://github.com/user-attachments/assets/346071cd-c38b-4e5f-a90b-39b70f2f7e9c)

**Comment by ghostinhershell** [comment]
Hi @EdmundDantes1, are you still running into this issue?

You can check our [GitHub Status Page](https://www.githubstatus.com/) to see for real-time and historical data on system performance + subscribe to issues for updates.

**Comment by chtholine** [comment]
Hello!

### Possible Solutions:

1. **Check if a pull request was created**  
   - Go to the repository where you're following the tutorial.
   - Click on the **Pull requests** tab.
   - If a pull request isn't there, manually create one by clicking **New pull request**.

2. **Ensure you've completed the previous steps correctly**  
   - If the tutorial expects changes to be made in a branch before opening a pull request, verify that you've committed those changes.
   - Make sure you've followed the instructions precisely.

3. **Restart the step**  
   - Try skipping the step and then redoing it.
   - Sometimes GitHub Learning Lab can be buggy, and refreshing or redoing the step can fix it.

4. **Check the "Details" link**  
   - Clicking on **Details** might give more specific information about what went wrong.

**Comment by EdmundDantes1** [comment]
Thank you 😊

**Reply by chtholine** [reply]
You're welcome! If my reply helped you, could you please mark my response as the answer? When you mark a reply as an answer, GitHub will highlight the comment which will help visitors quickly find the answer!

**Comment by 0PeterAdel** [comment]
# Troubleshooting Red Wavy Lines on Intro to GitHub

When the GitHub Actions workflow shows a “skipped” message or red wavy lines on a pull request, it usually indicates a small configuration issue. Here are some steps to check:

1. **Confirm You Have a Pull Request Open**  
   - Make sure you created a pull request from the correct branch.  
   - If the tutorial instructions say to create a pull request on a specific branch, confirm the branch name matches exactly.

2. **Check Workflow Triggers**  
   - Some GitHub Actions require certain triggers (like `pull_request` or `push` on a specific branch).  
   - Open the `.github/workflows/` folder to see if the YAML file has any conditions (like `on: pull_request` with specific branches).  
   - Verify the workflow is configured to run on the branch you’re using.

3. **Review YAML Syntax**  
   - Red wavy lines in a workflow file can signal a YAML formatting or indentation error.  
   - If the workflow file has errors, the action might skip or fail.  
   - Use an online YAML validator or check the “Actions” tab logs for clues.

4. **Check the “Actions” Tab**  
   - Navigate to the “Actions” tab in your repository to see if the workflow triggered.  
   - If it was skipped, there may be a note explaining why.

5. **Follow the Tutorial Steps Exactly**  
   - The “Intro to GitHub” tutorial often requires a specific file name or certain text in a file.  
   - Double-check that file names and instructions match what the tutorial expects.

If the issue persists after checking these points, consider sharing a link to the repository or screenshots of the workflow logs for additional guidance.
